flexget/tests/test_rtorrent.py

```diff
@@ -67,7 +67,9 @@ def test_load(self, mocked_proxy):
 
         fields = [p for p in called_args[2:]]
         assert len(fields) == 3
-        assert 'd.directory.set=\\/data\\/downloads' in fields
+        assert ('d.directory.set=\\/data\\/downloads' in fields
+                # Python 3.7+.
+                or 'd.directory.set=/data/downloads' in fields)
```

Comment:
Isn't that additional slash intentional ?

Reply:
In Python <= 3.6 more characters were escaped by re.escape(), including '/', but in Python >= 3.7 re.escape() only escapes regex metacharacters like '$' and '.'.  To make the test pass with Python >= 3.7 we need to look for `'d.directory.set=/data/downloads'`, with earlier versions we need to look for `'d.directory.set=\\/data\\/downloads'`.

I've pushed a new commit that expands the comment, let me know if it's clear enough or it needs more details.